### ISSUE TITLE
mousePressed not working on DOM buttons (p5.js 0.7.1)

### ISSUE THREAD

**Issue by tmarasco125** [processing/p5.js#3185]
#### Nature of issue?

- [x ] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [x ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [x ] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: 0.7.1
- Web browser and version: Chrome, vers. 68.0.3440.106 (Official Build) (64-bit)
- Operating System: Max osx 10.13.6
- Steps to reproduce this:

Hi! I just started using the newest version of p5.js and p5.dom.js, and the mousePressed() method throws an error when a DOM button is clicked. Here is my code:

```
var button;

function setup(){
noCanvas();
    button = createButton(" ");
    button.addClass("button button-primary button-border-thick button-box");
    button.mousePressed(logPress);
}

function draw(){


}

function logPress(){
console.log("Boing");
}
```
When the button is clicked, the console gives this error message:

_Uncaught TypeError: Cannot read property ‘_setProperty’ of undefined
at g.Element. (p5.min.js:7)
(anonymous) @ p5.min.js:7_

I was using version 0.7.1 of both p5.js and p5.dom.js, but when I switched p5 back to version 0.6.1, this worked perfectly. Has anyone else encountered this issue??

Here is my index.html code as well

```
<!-- Buttons core css -->
<html>
    <head>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.1/p5.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.1/addons/p5.dom.min.js"></script>

    </head>
<style>
    body {
    background-color: #FFFF00 
}


</style>
<body>
    <script src="code.js"></script>
    </body> 
<link rel="stylesheet" href="css/buttons.css">
</html>


```

**Comment by Spongman** [comment]
yes, this is indeed an issue in 0.7.1. it has been fixed in #3141 and will be in the next release.

**Comment by tmarasco125** [comment]
Thank you! Sorry for any double posting on the issue.

**Comment by Spongman** [comment]
no problem whatsoever.

**Comment by lmccart** [comment]
this should be fixed in the newest release 0.7.2
